### PR TITLE
po: Fix more Chinese translation

### PR DESCRIPTION
**Pull request by EricZhang456**
I blame all of these on fcitx5

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -495,7 +495,7 @@ msgstr "通用"
 
 #: src/widgets/shortcutsWindow.ui:19
 msgid "Show mnemonics"
-msgstr "显示助记符"
+msgstr "显示快速帮助" # I really don't know how to do this one. OG sounds weird.
```
Comment (tchx84) [inline po/zh_CN.po]: It's fine, but please remove the # comment

**Review by tchx84 — CHANGES_REQUESTED** [review]

**Review by tchx84 — APPROVED** [review]

**Comment by tchx84** [comment]
@EricZhang456 it seems this file is in conflict with the current version of the file, can rebase your changes on top of the latest version of this file?

**Comment by EricZhang456** [comment]
Huh. GitHub doesn't report any conflicts

**Comment by tchx84** [comment]
image

:cry:

**Comment by tchx84** [comment]
nvm.. it seems squashing did work!

**Pull request merged**